src/visualizers-workbench/P5VisualizerTemplate.ts

```diff
@@ -0,0 +1,174 @@
+import {P5Visualizer} from '../visualizers/P5Visualizer'
+import {VisualizerExportModule} from '@/visualizers/VisualizerInterface'
+
+/** md
+# p5 Visualizer Template
+
+[image should go here]
```

Comment:
Do we want to have an actual image in this one, so it isn't modeling being lazy about images as I have sometimes been guilty of?

Reply:
I vote for an actual image.

Reply:
Done: I've added images to the p5 template visualizer and the Differences visualizer. I also cleaned up the documentation, moved the docblocks closer to the corresponding code, and made the styles of the two example visualizers more consistent.

Reply:
Very nice. Thank you for compensating for my laziness in never putting an image into Differences. :grateful: